index.html

```diff
@@ -678,8 +678,59 @@ <h4>Search</h4>
                     <li>Federated queries to other TDDs</li>
                     <li>Spatial and network-limited queries</li>
                     <li>Links</li>
-                    </ul>
-                </section>
+		    </ul>
+        The Directory has three search APIs: syntactic search with JSON Path [[JSON Path]] or XPath [[xpath-31]], and semantic search with SPARQL [[sparql11-overview]].
+		    <span class="rfc2119-assertion" id="tdd-search-apis">
+			The Directory MUST implement the syntactic search with JSON Path, SHOULD implement the syntactic search with XPath, and MAY implement semantic search with SPARQL.
+		    </span>
+        <p class="ednote" title="Normative JSON Block">Should we have the previous sentence splitted into three different assertions?.</p>
+    		
+        <section id="jsonpath-semantic" class="normative">
+          <h4>Syntactic search: JSONPath</h4>
+          <span class="rfc2119-assertion" id="tdd-search-jsonpath-method"> The API MUST allow searching TDs using an HTTP <code>GET</code> request.</span> 
+          <span class="rfc2119-assertion" id="tdd-search-jsonpath-parameter"> The request MUST contain a valid JSON Path [[JSON Path]] as searching parameter.</span>
+          <span class="rfc2119-assertion" id="tdd-search-jsonpath-response">  A successful response MUST have 200 (OK) status, contain <code>application/json</code> Content-Type header, and in the body a set of complete TDs or a set of TD elements.</span>
```

Comment:
As suggested in discovery call, we can define another global term to define "TD Fragments" (in contrast to Partial TDs). This PR refers to this as TD elements in few places. This can be added in a separate PR.

Reply:
I changed the term in commit d718c1868e4a604a176ea72629ec4f686f0341ab